cri-o/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM registry.fedoraproject.org/fedora:25
+
+ENV VERSION=0 RELEASE=1 ARCH=x86_64
+LABEL com.redhat.component="cri-o" \
+      name="$FGC/cri-o" \
+      version="$VERSION" \
+      release="$RELEASE.$DISTTAG" \
+      architecture="$ARCH" \
+      usage="atomic install --system --system-package=no crio && systemctl start crio" \
+      summary="The cri-o daemon as a system container." \
+      maintainer="Yu Qi Zhang <jzehrarnyg@gmail.com>" \
+      atomic.type="system"
+
+COPY tmpfiles.template config.json.template service.template /exports/
+
+RUN dnf install --setopt=tsflags=nodocs -y crio runc && dnf clean all && \
```

Comment:
there seems to be no need to install runc as cri-o uses a _bundled_ runc because it needs a specific runc version which might conflict with the runc version in the runc package.